user/tests/wiring/no_fixture/fastpin.cpp

```diff
@@ -0,0 +1,40 @@
+
+#include "application.h"
+#include "unit-test/unit-test.h"
+
+
+
+test(FASTPIN_01_MaxDuration_SetReset) {
+    // Attempt pinSetFast and pinResetFast numerous times and check speed.
+    // Duration went from 62 -> 24 ticks per set on Gen2 with this change list
+
+#if PLATFORM_ID == PLATFORM_ARGON || \
```

Comment:
Nit: `#if HAL_PLATFORM_GEN == 3` or `#if HAL_PLATFORM_NRF52840`. We add one more platform, forget to change this conditional and the test never runs.

Let's also add an `#else` block with `#error` so that the test fails for any new platforms and we can immediately see that it requires attention.